app/main.py

```diff
@@ -0,0 +1,187 @@
+import json
+import os
+import sys
+
+from pathlib import Path
+
+sys.path.append(os.getcwd())
+
+import src.configuration as cf
+import streamlit as st
+
+st.set_page_config(
+    page_title="MyConfig",
+    page_icon="🖥️",
+    layout="wide",
+    initial_sidebar_state="auto",
+    menu_items={
+        "Get Help": "https://t.me/SuldinVyacheslav",
+        "Report a bug": "https://t.me/SuldinVyacheslav",
+        "About": "todo =)",
+    },
+)
+
+st.markdown(
+    Path(os.path.join(os.getcwd(), "css/header.md")).read_text(),
+    unsafe_allow_html=True,
+)
+
+
+if "configs" not in st.session_state:
+    st.session_state["configs"] = []
+
+
+def read_data(filename: str) -> dict:
+    f = open(filename)
+    r = json.loads(f.read())
+    f.close()
+    return r
```

Comment:
There is no need for a separate function. 
```python
with open(filename) as f:
    r = json.loads(f.read())
```

Reply:
Looks good, rewritten.